File: 30-Days-of-Code/Day-2-Operators/CSharp/solution.cs
using System;
using System.Collections.Generic;
using System.IO;

class Solution {
    static void Main(String[] args) {
        double mealCost;
        int tipPercent;
        int taxPercent;
        
        mealCost = double.Parse(Console.ReadLine());
        tipPercent = int.Parse(Console.ReadLine());
        taxPercent = int.Parse(Console.ReadLine());
        
        double tip = mealCost * ((double)tipPercent/((double)100));
        double tax = mealCost * ((double)taxPercent/((double)100));
        double totalCost = tip + tax + mealCost;
        
        Console.WriteLine("The total meal cost is {0:F0} dollars.", totalCost);
    }
}

File: 30-Days-of-Code/Day-2-Operators/FSharp/solution.fs
open System

[<EntryPoint>]
let main argv = 
    let (_, mealCost) = Double.TryParse(Console.ReadLine())
    let (_, tipPercent) = Int32.TryParse(Console.ReadLine())
    let (_, taxPercent) = Int32.TryParse(Console.ReadLine())
    
    let tip = mealCost * ((float) tipPercent / 100.0)
    let tax = mealCost * ((float) taxPercent / 100.0)
    
    let totalCost = mealCost + tip + tax
    
    let roundedTotalCost = Math.Round(totalCost)
    
    Console.WriteLine("The total meal cost is {0} dollars.", roundedTotalCost)
    
    0 
